### PR TITLE
92 point eip places

### PR DESCRIPTION
**Pull request by thullSL**

### CODE REVIEW

File: dataCenter/singleEcs.json [hunk 1]
```diff
@@ -1,5 +1,12 @@
 {
     "AWSTemplateFormatVersion": "2010-09-09",
+    "Parameters" : {
+        "PublicElasticIP" : {
+            "Type" : "String",
+            "Default" : "52.50.88.117",
+            "Description" : "The Ealstic IP to associate with this stack"
```
Comment (lclarksl) [inline dataCenter/singleEcs.json]: Elastic

File: dataCenter/singleEcs.json [hunk 2]
```diff
@@ -76,7 +83,7 @@
             "Properties" : {
                 "ContainerDefinitions" : [
                     {
-                        "Name" : "speakers",
+                        "Name" : "speakercomms",
```
Comment (lclarksl) [inline dataCenter/singleEcs.json]: speakerComms?

**Comment by jebbinSCL** [comment]
What's the summary of what this pull request is doing?

**Comment by thullSL** [comment]
It points an elastic IP the cloud formation stack

**Comment by jebbinSCL** [comment]
:shipit:

**Comment by dwybourn** [comment]
:shipit:

**Pull request merged**